yaml2rst.py

```diff
@@ -101,6 +101,9 @@ def convert(lines):
         elif line.startswith('# ') or line == '#':
             if state != STATE_TEXT:
                 yield ''
+            # Filter out [[[\d and ]]] at the end of a documentation line.
+            # Those sequences can be used for folding sections.
+            line = re.sub(r"\s*(:?\[{3}|\]{3})\d?$", "", line)
```

Comment:
> This would strip foldmarks in every line, not only in rst-comments

This is not true. Check the code.

> \s* is of no use?

It will additionally strip any trailing whitespace from the output file.
Trailing whitespace is considered bad practice.

> What should the trailing digit be? (Please document)

I documented that using an example in: https://github.com/ypid/yaml2rst/blob/feature/fold-marks/docs/fold-markers.rst
But in short. The digit can be used to put the fold on a certain level. If you avoid the explicit level, you will have to close the folds with the corresponding closing marker.